src/main/java/com/google/cloud/spanner/jdbc/JdbcConnection.java

```diff
@@ -402,6 +404,69 @@ public String getSchema() throws SQLException {
     return "";
   }
 
+  @Override
+  public SavepointSupport getSavepointSupport() {
+    return getSpannerConnection().getSavepointSupport();
```

Comment:
Have we intentionally skipped `checkClosed();` validation in this method? If yes, I didn't get why we won't need the check here.

Reply:
Not really intentionally. Added it to be consistent with the other methods.